gnmvidispine/vs_metadata.py

```diff
@@ -55,12 +55,12 @@ def toXML(self,mdGroup=None):
                 if isinstance(line,datetime):
                     line = line.strftime("%Y-%m-%dT%H:%M:%S%Z")
 
-                valueEl.text = unicode(line)
+                valueEl.text = str(line)
                 fieldEl.append(valueEl)
 
             timespanEl.append(fieldEl)
 
-        return ET.tostring(rootEl,encoding="UTF-8")
+        return ET.tostring(rootEl,encoding="utf8").decode("utf8")
```

Comment:
again, do we _want_ a bytestring here? I dunno, you're more familiar with the code than me at this point?

Reply:
Yes.